zxcvbn-ruby.gemspec

```diff
@@ -16,6 +16,8 @@ Gem::Specification.new do |gem|
   gem.version       = Zxcvbn::VERSION
   gem.license       = 'MIT'
 
+  gem.required_ruby_version = '~> 2.3'
```

Comment:
@johnsyweb - so I picked 2.3 do you think thats ok? or should it be 2.0?

Reply:
ah just read your full comment.. yeah a change log would be a great idea for this.. I'll pop it in here as well. Thanks!